sponsor.md

```diff
@@ -129,19 +129,27 @@ If you're interested in being vetted for this list, please reach out.
 
 - [Patreon](https://www.patreon.com/japaric/)
 - [GitHub](https://github.com/japaric), [email](mailto:jorge@japaric.io)
-- Open to direct contract work
 
 ### Details
 
 I want Rust to become a major player in the field of embedded systems as I believe it holds so much potential to improve the status quo. To that effect, I have been doing a bunch of groundwork in this area. Here's some of my recent work:
 
-- [Xargo](https://github.com/japaric/xargo), a Cargo wrapper for `no-std` development.
+- Lead of the [embedded devices WG](https://github.com/rust-lang-nursery/embedded-wg). This WG is an official part of the Rust project and we are working towards making embedded Rust a first class citizen. Check our [goals for the 2018 edition](https://github.com/rust-lang-nursery/embedded-wg/issues?q=is%3Aopen+is%3Aissue+milestone%3A%222018+edition%22).
+- [Embedded Rust on stable](https://github.com/rust-lang-nursery/embedded-wg/issues/42)
+- [Xargo](https://github.com/japaric/xargo), a Cargo wrapper for `no-std` development. Used for embedded, OS and game development! Notable users: [Baidu](https://github.com/baidu/rust-sgx-sdk), [Chucklefish](https://www.reddit.com/r/rust/comments/78bowa/hey_this_is_kyren_from_chucklefish_we_make_and/) and [Redox OS](https://www.redox-os.org/).
```

Comment:
@japaric you could probably also add `cross` in there, used by Microsoft Azure's [iotedge](https://github.com/Azure/iotedge/blob/master/edgelet/Cross.toml) project.

Reply:
:+1: